fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -495,20 +496,100 @@ void FixedLagSmoother::transactionCallback(
   }
 }
 
-void FixedLagSmoother::setDiagnostics(diagnostic_updater::DiagnosticStatusWrapper& status)
```

Comment:
The *diff* is a bit weird here. I simply added some helper functions that are used below.